SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -857,6 +865,44 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+void FSpatialGDKEditorToolbarModule::UpdateMobileClient() const
+{
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	const UIOSRuntimeSettings* IOSRuntimeSettings = GetDefault<UIOSRuntimeSettings>();
+	const FString ProjectName = FApp::GetProjectName();
+
+	const FString CommandLineArgs = FString::Printf(TEXT( "../../../%s/%s.uproject %s -game -log -workerType UnrealClient"), *ProjectName, *ProjectName, *(SpatialGDKSettings->ExposedRuntimeIP));
```

Comment:
If the user for some reason changes their client worker name from `UnrealClient` to something else, this will crash and burn right?

Reply:
oh yes. Where do we specify the name of the client worker?

Reply:
We do have 
```
namespace SpatialConstants
{
	static const FName DefaultServerWorkerType = TEXT("UnrealWorker");
	static const FName DefaultClientWorkerType = TEXT("UnrealClient");
}
``` but the user can override this somewhere in their config settings. I think there are helpers. @mattyoung-improbable ?

Reply:
@mattyoung-improbable

Reply:
@mattyoung-improbable ?

Reply:
hm... just took a look and I can only find an override for the server worker in Unreal. As soon as you connect as a client it seems to take the default client worker type.
@mattyoung-improbable  is there anything I'm missing?

Reply:
Pretty sure you can't change `UnrealClient`, just additional server workers